content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
+
+```raw title="symbolic form"
+10 + 32
+```
+
+In symbolic form, `+` is an operator: an **operator** is a symbol that is used to represent some kind of operation. In this example, `+` is used to represent the add operation “add the 2 numbers together”.  
+The actual combination of symbols `10 + 32` we say is an **expression**.
+
+{{<note title="Definition" type="info">}}
+An **expression** is a value or any valid combination of values and symbols that results in a single value.
+
+We say that expressions **evaluate** to a single value.
+
+So we say that `10 + 32` **evaluates** to the value `42`.
+
+{{</note >}}
+
+### Evaluating expressions
+
+We can take an expression like `36 * 45` and ask what it evaluates to. If we understand what the `*` operator represents ( in this case multiplication ) and if we understand some basic rules of arithmetic we can evaluate this expression.
+
+However, we can use computers to evaluate expressions.
+
+**NodeJS** is an application that allows us to run JavaScript programs. In other words, NodeJS can understand and execute programs written in the JavaScript language.
+
+NodeJS can be used in a number of different ways; however, one particular mode is the **REPL**.
+
+{{<note title="Definition" type="info">}}
+
+ℹ️ Definition: REPL is a special type of program that stands for
+
+**Read** - Users enter an expression that Node will **read**
+**Evaluate** - Node will then **evaluate** this expression
+**Print** - Node will **print** the result to the terminal
+**Loop** - Node will **loop** back to the beginning and prompt users for an input
+
+{{</note>}}
+
+The Node REPL enables us to input JavaScript instructions that are then executed by NodeJS. The REPL will then print out the result of this execution.
+
+We can type each of the following expressions into the REPL and then press enter to check the result.
+
+```bash
+10 + 32
+```
+
+```bash
+32 / 10
+```
+
+```bash
+3 ** 4
+```
+
+{{<note title="Activity" type="activity">}}
+
+In this activity, you'll check that you're set up to use the Node REPL on your machine.
+Try and complete each step below:
+
+1. Start a terminal application on your computer
+2. Check you've got Node installed on your computer
+3. Start the Node REPL in your terminal
+4. Enter the expressions above and evaluate them using the Node REPL
+
+Note: If you don't know how to do any of the steps above, then try searching for an appropriate command online - searching for things when you're stuck is super important part of being a developer!
+
+{{</note>}}
+
+{{<note title="Activity" type="activity">}}
+
+Try creating your own expressions and entering them into the Node REPL. In each case, before hitting enter, try predicting what the output will be in the REPL.
+
+{{</note>}}
+
+### 🏷️ Saving expressions
+
+In programming we often want to reuse particular expressions.
+Therefore, we need a way to refer back to particular expressions. Let’s consider the following string:
+
+```js
+"Hello there";
```

Comment:
I've noticed we're being pretty inconsistent with the use or not use of `;`s in code - do we want to try to be more consistent?

Reply:
Yeah I'll add some additional `;` for consistency throughout